dnf-behave-tests/features/transaction-sr/store.feature

```diff
@@ -58,7 +58,7 @@ Scenario: Store a transaction with an invalid transaction ID
  Then the exit code is 1
   And stderr is
       """
-      Error: Transaction ID "2" not found.
+      Not found given transaction ID
```

Comment:
This is clearly a regression in error message quality :slightly_smiling_face:

Reply:
Fixed